Maestro/subscriptions.json

```diff
@@ -1567,6 +1573,25 @@
           "PB_SymbolExpirationInDays": "10950"
         }
       }
+    },
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/corefx/tree/master/src/Common/src/CoreLib/**/*",
+        "https://github.com/dotnet/corefx/tree/release/2.1/src/Common/src/CoreLib/**/*",
```

Comment:
You should probably make sure the tool works well against master branches before enabling your code changes against release branches. (Not sure whether that means the release branches can go here and filtering is done further down the chain)

Reply:
>  Not sure whether that means the release branches can go here and filtering is done further down the chain


Yes, the filtering is done in the mirroring tool. The  purpose of adding them here is too just have a record of commits that went to the shared path in the master and release branch. 
we can enable the tool for release branches later and it will pickup all the missing changes from this record